VikingQuiz.Api/Controllers/FacebookController.cs

```diff
@@ -12,30 +12,29 @@ namespace vikingquiz.api.controllers
     [Route("api/[controller]")]
     public class FacebookController : Controller
     {
-        private readonly AuthenticationService authService;
+        private readonly AuthenticationService authenticationService;
         private readonly UserRepo repo;
```

Comment:
UserRepo need corect name like UserRepository, and parameter name not repo, more specific